mixins/form.jsx

```diff
@@ -306,9 +307,13 @@ module.exports = {
     var self = this;
     var props = {};
     fields.forEach(function(name) {
-      var prop = self.state[name].state.values;
+      var state = self.state[name].state;
+      var prop;
       // Currently some fields expose their values on the form, and not themselves.
       // So we need to check for props in both places until all field componenets are updated.
+      if (state) {
```

Comment:
State may or may not be defined on a component.

example, amount component has a state object, and frequency does not. Neither of them have a values object though.